charts/admission-controller/values.yaml

```diff
@@ -71,6 +73,12 @@ sysdig:
   # Overrides the `global.sysdig.secureAPIToken` parameter.
   secureAPIToken: ""
 
+  # **required for KSPM AC**
+  # <br/>Access Key to access Sysdig Secure.
+  # <br/><br/>Either this value or `sysdig.existingAccessKeySecret` is required
+  # Overrides `global.sysdig.accessKey`
+  accessKey: ""
```

Comment:
Isn't it the same as `secureAPIToken`?

Reply:
I don't think so. check: https://sysdig.atlassian.net/wiki/spaces/CS/pages/2945319073/Glia+Admission+Controller+Implementation+Use+Case

Reply:
No. AccessKey is the "agent installation access key", while the API token is the user or service account token. There were requirements to use access key instead of API token for cluster components, and that's why the new AC uses access key (like agent and the rest of components)